docs/ref/django-admin.txt

```diff
@@ -649,6 +649,11 @@ Providing one or more app names as arguments will limit the migrations created
 to the app(s) specified and any dependencies needed (the table at the other end
 of a ``ForeignKey``, for example).
 
+.. versionchanged:: 1.9
```

Comment:
`--noinput` is already specified as a common option (https://docs.djangoproject.com/en/1.7/ref/django-admin/#common-options), so that would create a conflict.